ocs_ci/ocs/resources/pvc.py

```diff
@@ -131,3 +131,28 @@ def create_multiple_pvc(number_of_pvc=1, pvc_data=None):
         pvc_objs.append(pvc_obj)
         log.info(f'Created Persistent Volume Claim {pvc_name}')
     return pvc_objs
+
+
+def get_pv_for_pvc(pvc_obj):
+    """
+    Function to return a PV object for the given PVC
+    Args:
+        pvc_obj (OCS): OCS object representing PVC
```

Comment:
why not PVC object?
With that you can use pvc_obj.backed_pv to get the name of that PV

Reply:
Done. Now using PVC object and pvc_obj.backed_pv to get the name of pv